pom.xml

```diff
@@ -342,6 +369,33 @@
                                 </rules>
                             </configuration>
                         </execution>
+                        <execution>
+                            <id>enforce-java-version</id>
+                            <goals>
+                                <goal>enforce</goal>
+                            </goals>
+                            <configuration>
+                                <rules>
+                                    <requireJavaVersion>
+                                        <version>[11,)</version>
+                                    </requireJavaVersion>
+                                </rules>
+                            </configuration>
+                        </execution>
+                        <execution>
```

Comment:
Since no dependencies are configured here, we can drop this configuration

Reply:
execution id: enforce-banned-dependencies

Reply:
Removed, commited and pushed.